### ISSUE TITLE
Enchanting Plus mod Exploit

### ISSUE THREAD

**Issue by superkitten12** [xJon/The-1.12.2-Pack#479]
**Basic details**
1. Are you using a legitimate launcher and Minecraft account:
**yes**.
2. Are you using a computer with 8GB or more of RAM? ([How to check](https://www.wikihow.com/Check-Computer-RAM)):
**yes**.
3. Are you using Java 8, 64-bit? ([How to check](https://www.wikihow.com/Determine-Java-Version)):
**yes**.

**Information required**
1. Is the modpack crashing: **no**
3. Is Optifine installed or any other additional mods: **no**

**Describe the issue**

The Enchanting Plus enchanting table can very easily provide infinite free enchantments of any degree if filled with at least 17(more will just make the price preview spit out a huge negative number, but you don't actually gain exp from it) of the Enchanting Plus books (the ones made from an enchanted book and 4 glowstone). This is already OP in itself, but I also happen to have an Enervation Dynamo, which with the Disjunctive Extraction specialization allows it to produce insane amounts of energy from items that are enchanted as much as possible(because the enchantments are free). Also while i was messing around with the enchanting plus table getting it set up and whatnot I found out the apparently it does not work with Magical Wood and cannot enchant books with enchantments other than Reading and Soulbound(both Ender IO and CoFH ones) and just enchants the book like a normal item and does not make an enchanted book. This can be worked around with the enchantment extractor but its kinda annoying.

tl;dr oil and uranium is worthless because wooden swords can easily make almost 5 million RF a pop.
also sorry for parentheses I didn't know how to fit that info in better lol

**Comment by xJon** [comment]
I was not able to reproduce what you described. Placing Enchanting Plus Books nearby acted like placing bookshelves, as it did not provide free enchantments & discounts.

**Comment by superkitten12** [comment]
Hmmm I'll get back to you then with some screenshots of my 'setup', didnt think I would need to at first since it worked even after reassembling to test the exact amount needed.

**Comment by superkitten12** [comment]
Okay here is all the information that I thought would be relevant, showing each part of the process. I also hovered over the book to make sure we were thinking of the same one since there are a bunch of different Enchanting Plus books (only one made with glowstone though).
[enchantmentexploit.zip](https://github.com/xJon/The-1.12.2-Pack/files/4966827/enchantmentexploit.zip)

**Comment by xJon** [comment]
@superkitten12 Can you upload the screenshots to a website like https://imgur.com?

**Comment by superkitten12** [comment]
Okay sorry about uploading in .zip the first time I just assumed that was the optimal way to submit multiple images. I now have an imgur account so here is the link with images https://imgur.com/a/hzWiM6H
I also added some extra description in the caption for each image.

**Comment by xJon** [comment]
Thank you for the report - I managed to reproduce it myself.
I've forwarded the issue to the mod author over https://github.com/Epoxide-Software/Enchanting-Plus/issues/175, although I'm afraid it won't be fixed anytime soon as this mod is not under active development for a year and a half now
